index.js

```diff
@@ -0,0 +1,177 @@
+
+let listOfWords = allTheWords.split(", ");
+let wordOfTheGame = listOfWords[Math.floor( (Math.random() * listOfWords.length) + 1 )];
+console.log("Word of the game is: " + wordOfTheGame);
+let dashes = document.querySelector("#dashes");
+let newNode = document.createElement("p");
+let newChar = ""; //change to string dashes
+for(let i = 0; i < wordOfTheGame.length; i++) {
+    newChar += "_ ";
+}
+newNode.innerText = newChar;
+dashes.parentNode.replaceChild(newNode, dashes);
+let typeOfGuess = document.querySelector("#typeOfGuess");
+let theButton = document.querySelector("#submitButton");
+let input = document.querySelector("#userInput"); //grab the input
+let arrayOfDashesToBeChanged = (newChar.split(" "));
+newChar = newChar.trim();
+let amountOfGuesses = document.querySelector("#amountOfGuesses");
```

Comment:
You could combine amountOfGuesses with startingNumberForAmountOfGuesses and just make one variable called "lives" or "guesses".